system/Controller.php

```diff
@@ -152,16 +160,22 @@ protected function validateData(array $data, $rules, array $messages = [], ?stri
         return $this->validator->run($data, null, $dbGroup);
     }
 
+    protected function setConfigValidator(Validation $config): void
+    {
+        $this->configValidation = $config;
+    }
```

Comment:
Do we really need this method?
If there is a protected property, we can set it in our controllers.